blockstack/blockstackd.py

```diff
@@ -3795,6 +3795,19 @@ class Program {
             hash_out.write(json_hash)
             hash_out.flush()
 
+        placeholder_dict = {
+            'missing-A-0b769752920b30884d6d68bd77ee0311a99505301a973639ff390258a6b5b7bd': 'SM37EFPD9ZVR3YRJE7673MJ3W0T350JM1HVZVCDC3',
+            'missing-A-65dfbb5430f1c0e778c800e85c64a038f66e298c358024025a23af686e1fd040': 'SM28P04DXXDEY3WY02VQJD4TS7VZS00Z6SX54XH5J',
+            'missing-A-d7d46c855cf2912fd99c8ae5ec2ef3b5d5d6a460a22b46eee0f0ef2a9789ff8b': 'SM1ZH700J7CEDSEHM5AJ4C4MKKWNESTS35DD3SZM5',
+            'missing-A-ee397a3bbc5e70cb6a19e14ccdb935f165dd658707bf0ec4c936da7e8f6f1000': 'SM260QHD6ZM2KKPBKZB8PFE5XWP0MHSKTD1B7BHYR',
+            'not_distributed_2ccfda6ed403c416b9749e513a31d36717e33f5bdbb11c51f3a22fb9eb336443': 'SM65WYACEEX6KBM3B27XX0DE1NAPHT5WJ9D2B1EY',
+            'not_distributed_88a422b7ccaa54e5d8225d5130dc562ffdc57cc54bf274f2b631061ae86d0dc0': 'SM1TP7CNY63KQY7DVRDVPNE1X73ND3PT8JX46971V',
+            'not_distributed_8941f1658fc78107ad34cf4d8e96a920a5a3257384188fb21e701c5e6a4b81fb': 'SM2ZP9PPEHXGPAAMYRJNMPVET8WRKBSYY1AZ8SWHP',
+            'not_distributed_d6d337469b6283046ed909a03a1f45d862d6c1361d5d0ee56ae660e25866b4c4': 'SM3TJ7J6DR9KFQXQGK9SY7M7AEJT33C8NV639J99C',
```

Comment:
@jessesoslow can you confirm that this is the right/expected set of placeholder address mappings?
Related to Jude's request https://github.com/blockstack/stacks-blockchain/issues/2106#issuecomment-738968203